ls1intum/Artemis
# `General`: Use short result string as default

### Checklist
#### General


#### Client

### Motivation and Context


The result string can take up much space, especially for programming exercises. Which is the reason for a short form (only achieved points).

### Description

This short form is now used as the default for results. Only the tutor assessment for programming exercises directly shows the passed tests to help the tutor quickly see this information when assessing or replying to a complaint.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Tutor
- 1 Programming Exercise with some submissions

1. Log in to Artemis
2. Check that the shot result string is used in almost all places (e.g. scores table, student view, ...)
3. Check that the tutor sees the relevant information quickly

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Screenshots


Current long format (short format would omit tests and issues here):
![Bildschirmfoto 2023-04-27 um 15 21 50](https://user-images.githubusercontent.com/38322605/234874946-7cfbd93a-cfe8-40c3-97f3-4e4ff5e02b2c.png)
